src/detection_linux.cpp

```diff
@@ -91,14 +91,13 @@ void Stop() {
 
```

Comment:
> Furthermore, the tests don't all pass, but it appears they don't in the main repo either.

hmm, I think they should pass. Can someone else confirm this?

What's not passing about them?

Reply:
I had someone run this on their Linux machine it looks like one of the tests is failing on `master`.

```sh
$ npm test
...
Failures:
1) usb-detection can exit gracefully after `startMonitoring` then an async delayed `stopMonitoring`
  Message:
    Failed
  Stack:
    Error: Failed
           at <Jasmine>
           at /home/manjaro/node-usb-detection/test/test.js:157:11
           at processTicksAndRejections (node:internal/process/task_queues:96:5)

11 specs, 1 failure
```

As long as this PR has the same result, we can move forward with merging ⏩

Reply:
huh, must be a quirk of my VM. Also, @Costava did you really run that on a pinephone? :joy: 

@MadLittleMods is that sufficient testing info or would you like me to confirm it on my box as well?

Reply:
The errors I'm seeing are all Async timeouts, which may be what's up.

Reply:
> @MadLittleMods is that sufficient testing info or would you like me to confirm it on my box as well?

It's sufficient if we run it again with the new changes. Would be nice to get yours ✅ as well since it should be 🤔

> The errors I'm seeing are all Async timeouts, which may be what's up.

@umbernhard To be clear, the tests require manual interaction to succeed. The test output say what action is expected but just plug and unplug any USB device over and over while running the tests.

Reply:
oh heck, that didn't occur to me. I'll do that now

Reply:
yep, they're all green for me now.

Reply:
> Also, Costava did you really run that on a pinephone? :joy:

@umbernhard Yes, ran the tests on a physical pinephone :)